backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -1387,6 +1411,14 @@ def bind(self, configurator: Configurator) -> None:
         )
         configurator.add_view(self.join_workspace, route_name="post_user_workspace")
 
+        # user role
+        configurator.add_route(
+            "get_user_role_workspace",
+            "/users/{user_id:\d+}/role_workspaces",  # noqa: W605
```

Comment:
@lebouquetin hear is the route to review

Reply:
What type of object is returned? This is a list of roles or a list of workspaces? The route name is not clear

Reply:
this end point return a list of object with the user, the workspace and the role of the user in the workspace

Reply:
see comment on tracim about the naming

Reply:
The tracim thread: https://algoo.tracim.fr/ui/workspaces/165/contents/thread/134421